src/utils/FormattingUtils.ts

```diff
@@ -0,0 +1,165 @@
+}
+
+/**
+ * Converts a stringified number into a BigNumber. If the string version of the num contains a `.` then it is a number which needs to be parsed to a string int.
+ * @param num The number to parse.
+ * @returns The parsed BigNumber.
+ * @note This is a temporary function until we can backport support for decimal points to @across-protocol/sdk-v2.
+ */
+export const toBN = (num: BigNumberish): BN => {
+  // If the string version of the num contains a `.` then it is a number which needs to be parsed to a string int.
+  if (num.toString().includes(".")) {
+    return BigNumber.from(parseInt(num.toString()));
+  }
+  return BigNumber.from(num.toString());
+};
+
+export const formatFeePct = (relayerFeePct: BigNumber): string => {
+  // 1e18 = 100% so 1e16 = 1%.
+  return createFormatFunction(2, 4, false, 16)(toBN(relayerFeePct).toString());
+};
+
+/**
+ * Creates an etherscan link for a transaction or address on a given network.
+ * @param txHashOrAddress The transaction hash or address to link to.
+ * @param chainId The chainId to link to.
+ * @returns The etherscan link. This link will be formatted in markdown.
+ */
+export function etherscanLink(txHashOrAddress: string, chainId: number | string): string {
+  return _createEtherscanLinkMarkdown(txHashOrAddress, Number(chainId)) ?? "<>";
+}
+
+/**
+ * Resolves a domain to an etherscan link for a given network.
+ * @param networkId The network to link to.
+ * @returns The etherscan link. If the networkId is not supported, the default etherscan mainnet link will be returned.
+ */
+export function createEtherscanLinkFromTx(networkId: number): string {
+  return PUBLIC_NETWORKS[networkId]?.etherscan ?? DEFAULT_BLOCKCHAIN_EXPLORER_DOMAIN;
+}
+
+/**
+ * Shortens a lengthy hexadecimal string to a shorter version with an ellipsis in the middle.
+ * @param hex A hexadecimal string to be shortened.
+ * @returns The shortened hexadecimal string.
+ * @example shortenHexString("0x772871a444c6e4e9903d8533a5a13101b74037158123e6709470f0afbf6e7d94") -> "0x7787...7d94"
+ */
+export function createShortHexString(hex: string): string {
+  return hex.substring(0, 5) + "..." + hex.substring(hex.length - 6, hex.length);
+}
+
+/**
+ * Generates a valid etherscan link for a given transaction hash or address.
+ * @param hex The transaction hash or address to link to.
+ * @param chainId The chainId to link to.
+ * @returns A formatted markdown etherscan link to the given transaction hash or address on the given chainId.
+ */
+function _createEtherscanLinkMarkdown(hex: string, chainId = 1): string | null {
+  if (hex.substring(0, 2) != "0x") {
+    return null;
+  }
+  const shortURLString = createShortHexString(hex);
+  // Transaction hash
+  if (hex.length == 66) {
+    return `<${createEtherscanLinkFromTx(chainId)}tx/${hex}|${shortURLString}>`;
+  }
+  // Account
+  else if (hex.length == 42) {
+    return `<${createEtherscanLinkFromTx(chainId)}address/${hex}|${shortURLString}>`;
```

Comment:
I'm a firm believer that we should be applying the trailing / here, since the hostname is totally separate from the subsequent path/resource.

```suggestion
    return `<${createEtherscanLinkFromTx(chainId)}/tx/${hex}|${shortURLString}>`;
  }
  // Account
  else if (hex.length == 42) {
    return `<${createEtherscanLinkFromTx(chainId)}/address/${hex}|${shortURLString}>`;
```